src/snovault/elasticsearch/indexer.py

```diff
@@ -334,8 +338,99 @@ def _get_nodes(request, indexer_state):
     return this_node, other_node, continue_on, did_timeout
 
 
+class IndexerInfo(LocalStoreClient):
```

Comment:
I think we can move this class to a separate file, maybe the same file with `LocalStoreClient`